### ISSUE TITLE
Provide hint to setup heart rate in setup before being able to select it

### ISSUE THREAD

**Issue by piratenpanda** [jonasoreland/runnerup#302]
At the moment the heart rate goal is greyed out when you didn't set up your heart rate ranges with your age and gender. But it took me a while to find it and I thought it didn't work because my heart rate already showed and it was still greyed out.

**Comment by jerriclynsjohn** [comment]
Yeah I agree that part is really confusing, I'm still confused about the pairing procedure
